app/services/Worker.php

```diff
@@ -5,8 +5,10 @@
 use GanttDashboard\App\Models\Workers;
 use GanttDashboard\App\Validators\Worker as WorkerValidator;
 use Phalcon\Mvc\Model\ResultsetInterface;
-use Phalcon\Validation\Message\Group as MessageGroup;
 use Phalcon\Mvc\Model;
+use GanttDashboard\App\Services\WorkerProject as WorkerProjectService;
+use GanttDashboard\App\Services\History as HistoryService;
+use GanttDashboard\App\Validators\WorkerProject as WorkerProjectValidator;
 
 class Worker
```

Comment:
update your "getWorker" docblock and return type - the declaration of Phalcon\Mvc\Model is too general